features/users/profile_karma.feature

```diff
@@ -0,0 +1,23 @@
+@vcr
+Feature: As a user of the site
```

Comment:
critical thing here is that the first line of feature is the name of the feature, rather than the first line of the three component connextra format, i.e. 

```Feature: Status within the Community
As a member of the community
In order to see my status in the community
I want to see karma on my profile page
```

This is important because the VCR cache files will be created based on the words immediately after `Feature:`

Do also check to see if you have any new VCR cache files to check in ...